source/tools/entry.js

```diff
@@ -6,6 +6,16 @@ const EntryProperty = {
     Username:                     "username"
 };
 
+const EntryDefaultAttribute = {
+    PassModifiedTime:              "passModifiedTime",
+    PassExpiryTime:                "passExpiryTime",
+    PassExpiry:                    "passExpiry"
```

Comment:
User wont be reading these directly. We have frontends that handle these formattings :)